src/components/Button/Button.stories.tsx

```diff
@@ -12,39 +12,37 @@ const meta: Meta<typeof Button> = {
   title: "Components/Form Elements/Button",
   component: Button,
   argTypes: {
-    buttonText: {
-      description: "**Only used for Storybook – ** Set the button label text.",
+    isDisabled: { table: { defaultValue: { summary: "false" } } },
+    mouseDown: { table: { defaultValue: { summary: "false" } } },
+    size: {
+      control: { type: "radio" },
+      options: sizesArray,
+      table: { defaultValue: { summary: "medium" } },
     },
     variant: {
       control: { type: "select" },
       options: buttonVariantsArray,
       table: { defaultValue: { summary: "primary" } },
     },
+    buttonText: {
+      description: "**Only used for Storybook – ** Set the button label text.",
+      table: { category: "Storybook Only" },
```

Comment:
This is a nice new feature:

<img width="667" height="481" alt="Image" src="https://github.com/user-attachments/assets/b0ef7f93-f7d0-49a8-a152-dbe458549451" />

Reply:
No... doesn't look like there's an option to hide/collapse by default.

Reply:
Womp. Ok thanks for looking into!